docs/.vitepress/config.mts

```diff
@@ -0,0 +1,65 @@
+import { defineConfig } from 'vitepress'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: "JECS",
+  description: "Just another ECS",
```

Comment:
You're going to need to add this line to avoid problems in the future:

```ts
base: "/jecs/",
```

Why? Because if you don't, the pages will try to build to `https://ukendio.github.io/` and break because it's not under jecs.